aliroot.sh

```diff
@@ -14,6 +14,14 @@ build_requires:
   - "Xcode:(osx.*)"
 env:
   ALICE_ROOT: "$ALIROOT_ROOT"
+overrides:
```

Comment:
@ktf , @pzhristov , 
This seems to be a wrong place where to override versions for building AliRoot for run2. Could you, please, tell me where these definitions should go? Thank you.

Reply:
The last build/AliRoot/alidist  (see the log below) now fails because the new versions are not compatible with ROOT 5.x. So we need to keep the old versions for building AliRoot. However, as I asked above, I didn't find where these should be declared. @pzhristov , could you help with this? Thank you.